### PR TITLE
Add REST API documentation

### PR DESCRIPTION
**Pull request by kklocek**

### CODE REVIEW

**Comment by kklocek** [comment]
There is additional task to do in this PR - update Swagger metadata.
I will handle this.

File: src/amoc_api_scenario_handler.erl [hunk 1]
```diff
@@ -21,18 +21,75 @@
...
+
+    ResponseBodyGet =
+    #{<<"200">> =>
+      #{description => <<"response object">>,
+        schema =>
+        #{type => <<"object">>,
+          required => [<<"scenario_status">>],
+          properties =>
+          #{scenario_status => #{<<"type">> => <<"string">>,
+                                 <<"description">> => <<"loaded | running | finished">>
```
Comment (elvisci) [inline src/amoc_api_scenario_handler.erl]: Line 55 is too long:                                  <<"description">> => <<"loaded | running | finished">>.

File: src/amoc_api_scenarios_handler.erl [hunk 2]
```diff
@@ -17,18 +17,71 @@
...
+    #{name => <<"request body">>,
+      in => body,
+      description => <<"request body (as json)">>,
+      required => true,
+      schema =>
+      #{type => <<"object">>,
+        required => [<<"scenario">>, <<"module_source">>],
+        properties =>
+        #{scenario => #{<<"type">> => <<"string">>,
+                        <<"description">> => <<"Name of scenario">>
```
Comment (arkgil) [inline src/amoc_api_scenarios_handler.erl]: Documentation should explain if `scenario` is a module name or a file name ending with `.erl`.

**Comment by arkgil** [comment]
@michalwski looks good to me. Can you please review it once more?

File: src/amoc_api_scenario_handler.erl [hunk 3]
```diff
@@ -21,18 +21,76 @@
...
+    #{name => <<"request body">>,
+      in => body,
+      description => <<"request body (as json)">>,
+      required => true,
+      schema =>
+      #{type => <<"object">>,
+        required => [<<"users">>],
+        properties =>
+        #{users => #{<<"type">> => <<"integer">>,
+                           <<"description">> => <<"Number of users to start">>
```
Comment (elbrujohalcon) [inline src/amoc_api_scenario_handler.erl]: Indentation looks a little skewed here

**Comment by michalwski** [comment]
Looks good to me. Please fix the indentation mentioned by @elbrujohalcon and this PR can be merged.
@arkgil please merge it when fixed, I'll probably not be able to do it tomorrow.

**Pull request merged**